newIDE/app/src/Export/Builds/BuildCard.js

```diff
@@ -64,46 +116,266 @@ export const BuildCard = ({
+                      <Line noMargin expand>
+                        <TextField
+                          ref={nameInput}
+                          style={styles.textField}
+                          value={name}
+                          margin="none"
+                          onChange={(_, value) => setName(value)}
+                          onBlur={() => {
+                            onBlurEditName(i18n);
+                          }}
+                          hintText={buildName}
+                          disabled={gameUpdating}
+                          onKeyPress={event => {
+                            if (shouldValidate(event) && nameInput.current)
+                              nameInput.current.blur();
+                          }}
+                          maxLength={50}
+                        />
+                        {gameUpdating && (
+                          <>
+                            <Spacer />
+                            <CircularProgress style={styles.circularProgress} />
+                          </>
+                        )}
+                      </Line>
+                    ) : (
+                      <Line noMargin alignItems="baseline">
+                        <Text noMargin>{buildName}</Text>
+                      </Line>
+                    )}
+                  </Line>
+                  <Line noMargin alignItems="center">
+                    <BackgroundText style={{ textAlign: 'left' }}>
+                      {build.id}
+                    </BackgroundText>
+                    <Spacer />
+                    <IconButton size="small" onClick={onCopyUuid}>
+                      <Copy style={styles.buildButtonIcon} />
+                    </IconButton>
+                  </Line>
+                  <LargeSpacer />
+                  <Line expand noMargin justifyContent="space-between">
+                    {!isOld && (
+                      <BuildProgressAndActions
+                        build={build}
+                        game={game}
+                        onGameUpdated={onGameUpdated}
+                        gameUpdating={gameUpdating}
+                        setGameUpdating={setGameUpdating}
+                        onCopyToClipboard={() => setShowCopiedInfoBar(true)}
+                      />
+                    )}
+                    {isOld && (
+                      <EmptyMessage>
+                        <Trans>
+                          This build is old and the generated games can't be
+                          downloaded anymore.
+                        </Trans>
+                      </EmptyMessage>
+                    )}
+                  </Line>
+                </Column>
+              </Card>
+              <InfoBar
+                visible={showCopiedInfoBar}
+                hide={() => setShowCopiedInfoBar(false)}
+                message={<Trans>Copied to clipboard!</Trans>}
+              />
+              {showConfirmationDialog && (
```

Comment:
I'm a bit skeptical about this. We already have `showConfirmDialog`. Either we decide to use `showConfirmDialog` everywhere, either we decide to migrate to something else, but we risk inconsistency in the user experience.

If we decide to have our own confirm dialogs, I would then have a `ConfirmDialog` component. Also make sure that the button to Cancel has the focus (so that it's just a matter of pressing Tab and Enter to validate).

Reply:
I actually did not know about this technology 😅